TS04-Plotting-Time-Series-R.Rmd

```diff
@@ -100,12 +103,12 @@ harMet15.09.11$datetime<-as.POSIXct(harMet15.09.11$datetime,
                     format = "%Y-%m-%dT%H:%M",
                     tz = "America/New_York")
 #daily HARV met data, 2009-2011
-harMetDaily.09.11 <- read.csv(file="AtmosData/HARV/hf001-06-daily-m.csv",
+harMetDaily.09.11 <- read.csv(file="Met_HARV_Daily_2009_2011.csv",
                      stringsAsFactors = FALSE)
 
 #covert date to Date type
 
-harMetDaily.09.11$date<-as.Date(harMetDaily.09.11$date)
+harMetDaily.09.11$date <- as.POSIXct(harMetDaily.09.11$date)
```

Comment:
This one had something to do with the plotting that followed. It didn't work for me with as.Date but it did with POSIX.


Marisa Guarinello
Environmental Data Manager
MILES EPSCoR Project
Northwest Knowledge Network
University of Idaho, Office of Research & Economic Development
mguarinello@uidaho.edu
208-885-5044


From: "Megan A. Jones" <notifications@github.com<mailto:notifications@github.com>>
Reply-To: data-lessons/NEON-R-Tabular-Time-Series <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, November 24, 2015 at 10:36 AM
To: data-lessons/NEON-R-Tabular-Time-Series <NEON-R-Tabular-Time-Series@noreply.github.com<mailto:NEON-R-Tabular-Time-Series@noreply.github.com>>
Cc: Marisa Guarinello <mguarinello@uidaho.edu<mailto:mguarinello@uidaho.edu>>
Subject: Re: [NEON-R-Tabular-Time-Series] from marisa edits to lessons fri nov 20 (#53)


In TS04-Plotting-Time-Series-R.Rmd<https://github.com/data-lessons/NEON-R-Tabular-Time-Series/pull/53#discussion_r45775234>:

>                       stringsAsFactors = FALSE)
>
>  #covert date to Date type
>
> -harMetDaily.09.11$date<-as.Date(harMetDaily.09.11$date)
> +harMetDaily.09.11$date <- as.POSIXct(harMetDaily.09.11$date)


@mguarinello<https://github.com/mguarinello> When dealing with date only and not time. Why use POSIXct instead of Date? Both were taught in these lessons (granted as.Date is in 00, not this one).

-
Reply to this email directly or view it on GitHub<https://github.com/data-lessons/NEON-R-Tabular-Time-Series/pull/53/files#r45775234>.